kimchi/src/circuits/argument.rs

```diff
@@ -3,6 +3,7 @@
 //! Both the permutation and the plookup arguments fit this type.
 //! Gates can be seen as filtered arguments,
 //! which apply only in some points (rows) of the domain.
+//! For more info, read book/kimchi/src/kimchi/arguments.md
```

Comment:
I think it's `book/src/kimchi/arguments.md` unless you're referring to something different